cmake/FindAutotalks.cmake

```diff
@@ -0,0 +1,91 @@
+        PATHS $ENV{HOME}
+        CMAKE_FIND_ROOT_PATH_BOTH # Seems like this is not needed in newer CMake
+    )
+
+    set(AUTOTALKS_INCLUDE_DIRS ${AUTOTALKS_ROOT}/include ${AUTOTALKS_ROOT}/ref_src/include ${AUTOTALKS_ROOT}/depend/device/include)
+    set(AUTOTALKS_LIBS_DIR ${AUTOTALKS_ROOT}/output.sec/x86/lib)
+    set(AUTOTALKS_OBJ_DIR ${AUTOTALKS_ROOT}/output.sec/x86/obj)
+
+
+    if (NOT TARGET Autotalks::AtlkRemote)
+        add_library(Autotalks::AtlkRemote UNKNOWN IMPORTED)
+        set_target_properties(Autotalks::AtlkRemote PROPERTIES
+            IMPORTED_LOCATION "${AUTOTALKS_LIBS_DIR}/libatlkremote_linux_u.so"
+            INTERFACE_INCLUDE_DIRECTORIES "${AUTOTALKS_INCLUDE_DIRS}")
+    endif()
+
+    if (NOT TARGET Autotalks::ToMcrypt)
+        add_library(Autotalks::ToMcrypt UNKNOWN IMPORTED)
+        set_target_properties(Autotalks::ToMcrypt PROPERTIES
+            IMPORTED_LOCATION "${AUTOTALKS_LIBS_DIR}/libtomcrypt.a"
+            INTERFACE_INCLUDE_DIRECTORIES "${AUTOTALKS_INCLUDE_DIRS}")
+    endif()
+
+    if (NOT TARGET Autotalks::ToMmath)
+        add_library(Autotalks::ToMmath UNKNOWN IMPORTED)
+        set_target_properties(Autotalks::ToMmath PROPERTIES
+            IMPORTED_LOCATION "${AUTOTALKS_LIBS_DIR}/libtommath.a"
+            INTERFACE_INCLUDE_DIRECTORIES "${AUTOTALKS_INCLUDE_DIRS}")
+    endif()
+
+    set(AUTOTALKS_EXECUTABLES ${AUTOTALKS_OBJ_DIR}/target_type/target_type_remote.o ${AUTOTALKS_OBJ_DIR}/platform_type/platform_type_linux_u.o ${AUTOTALKS_OBJ_DIR}/link_layer/linux_u/remote/link_layer_remote.o ${AUTOTALKS_OBJ_DIR}/link_layer/linux_u/ipc/link_layer_ipc.o ${AUTOTALKS_OBJ_DIR}/link_layer/linux_u/tee/link_layer_tee_stub.o ${AUTOTALKS_OBJ_DIR}/link_layer/link_layer_interface.o ${AUTOTALKS_OBJ_DIR}/flash_reader/linux_u/remote/flash_reader_remote.o ${AUTOTALKS_OBJ_DIR}/ref_sys/ref_sys.o ${AUTOTALKS_OBJ_DIR}/ref_sys/ref_sys_logger.o ${AUTOTALKS_OBJ_DIR}/log_reader/log_reader.o ${AUTOTALKS_OBJ_DIR}/secure_hdif_crypto_layer/crypto_layer_ref_sys.o ${AUTOTALKS_OBJ_DIR}/secure_hdif_crypto_layer/crypto_layer_interface.o ${AUTOTALKS_OBJ_DIR}/secure_hdif_crypto_layer/crypto_lib_wrapper.o ${AUTOTALKS_OBJ_DIR}/secure_storage/secure_storage.o ${AUTOTALKS_OBJ_DIR}/time_sync/time_sync.o ${AUTOTALKS_OBJ_DIR}/time_sync/time_sync_poti.o ${AUTOTALKS_OBJ_DIR}/poti_lib/gnss/gnss.o ${AUTOTALKS_OBJ_DIR}/poti_lib/nmea/nmea.o ${AUTOTALKS_OBJ_DIR}/poti_lib/nmea/nmea_io.o ${AUTOTALKS_OBJ_DIR}/poti_lib/nmea/nmea_io_check.o ${AUTOTALKS_OBJ_DIR}/poti_lib/nmea/nmea_nav.o ${AUTOTALKS_OBJ_DIR}/poti_lib/nmea/nmea_parse.o ${AUTOTALKS_OBJ_DIR}/poti_lib/gnss/teseo/teseo.o ${AUTOTALKS_OBJ_DIR}/poti_lib/gnss/teseo/teseo_commands.o ${AUTOTALKS_OBJ_DIR}/poti_lib/coord/coord.o ${AUTOTALKS_OBJ_DIR}/poti_lib/periodic_alarm/periodic_alarm.o ${AUTOTALKS_OBJ_DIR}/poti_lib/api/poti_api.o)
+
+else() # Craton device
+    find_path(
+        AUTOTALKS_ROOT
+        NAMES "include/atlk/v2x_service.h"
+        PATHS $ENV{HOME}
```

Comment:
So you expect the API to be located at */home/your_user/autotalks_craton_api*? Is this suggested by Autotalks?

Reply:
I expect the user to create symbolic link from _/home/your_user/autotalks_craton_api_ or _/home/your_user/autotalks_secton_api_ to SDK root, I thought that it was simpler as there does not seem to be any consistent path defined by autotalks. However, the SDK itself is found in `secton-sdk-5.14.0-rel_sec_USB` or `secton-sdk-5.14.0-rel_sec_USB` for version `5.14.0`, so maybe that could be used somehow?

Reply:
Would you please add a note about this expectation to *autotalks-sdk-build.md*?

Reply:
It was already there but I tried to make the explanation more clear